docs/workflows/verify-identity.http

```diff
@@ -0,0 +1,76 @@
+# @no-cookie-jar
```

Comment:
@kiootic Just curious how do you use this file with the env file?

Reply:
I use [httpyac](https://httpyac.github.io/) VS Code extension.

Reply:
Thanks!